specs/capella/light-client/full-node.md

```diff
@@ -76,3 +76,31 @@ def block_to_light_client_header(block: SignedBeaconBlock) -> LightClientHeader:
         execution_branch=execution_branch,
     )
 ```
+
+## Deriving light client data
+
+### `create_light_client_update`
+
+Full nodes SHOULD continue deriving the `LightClientUpdate` for each sync committee period as per the altair specs but with the modified or upgraded `attested_header` and `finalized_header` with the following considerations:
+
+- Both `attested_header` and `finalized_header` are of the same fork types as `LightClientUpdate` and might need upgradation to bundle as update.
```

Comment:
"upgradation" > "upgrading"?

Also, the `capella.block_to_light_client_header` function above works perfectly fine on altair blocks as well. So, whether an implementation first gets the original representation and upgrades from there, or whether it immediately computes the correct representation doesn't matter that much.